app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,5 @@
 <resources>
     <string name="app_name">KotlinSample</string>
+    <string name="hello_world">Hello World!</string>
+    <string name="hello_kotlin">Hello Kotlin!</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.hello_kotlin` appears to be unused
 Unused resources make applications larger and slow down builds.